### ISSUE TITLE
Machine learning/AI/Prob research and resources

### ISSUE THREAD

**Issue by tauoverpi** [vojes-enterprises/PreFOEP#46]
Consider the following resources for machine learning, probability, and classic AI

## tagged machine learning

[Wheel/Rail Adhesion State Identification of Heavy-Haul Locomotive Based on Particle Swarm Optimization and Kernel Extreme Learning Machine](https://www.hindawi.com/journals/jat/2020/8136939/)
[Baselines and Bigrams: Simple, Good Sentiment and Topic Classification](https://nlp.stanford.edu/pubs/sidaw12_simple_sentiment.pdf)
[Bayesian Methods for Support Vector Machines: Evidence and Predictive Class Probabilities](http://web.cs.iastate.edu/~honavar/bayes-svm.pdf)
[Deep Learning: A Critical Appraisal](https://arxiv.org/ftp/arxiv/papers/1801/1801.00631.pdf)
[ARTIFICIAL INTELLIGENCE 2E](https://artint.info/2e/html/ArtInt2e.html)
[Multi label Image Classification](https://suraj-deshmukh.github.io/Keras-Multi-Label-Image-Classification/)
[Finite and Infinite Machines by Marvin Minsky](https://github.com/media-lib/science_lib/raw/master/books/Computation_Finite_And_Infinite_Machines_by_Marvin_Minksy.pdf)
[Generalized Convolution and Efficient Language Recognition](https://arxiv.org/pdf/1903.10677.pdf)
[A Critical Review of Recurrent Neural Networks for Sequence Learning](https://arxiv.org/pdf/1506.00019.pdf)
[A JIT Compiler for Neural Network Inference](https://arxiv.org/pdf/1906.05737v1.pdf)
[Programming with a Differentiable Forth Interpreter](https://arxiv.org/pdf/1605.06640)
[Graph Neural Networks: A Review of Methods and Applications](https://arxiv.org/pdf/1812.08434.pdf)
[Machine Learning Classification over Encrypted Data](https://eprint.iacr.org/2014/331.pdf)

## tagged markov process

[Pairwise Markov Chain: A Task Scheduling](mason.gmu.edu/~hzhang18/publications/Conf_Paper_2.pdf)
[General state space Markov chains and MCMC algorithms*](https://arxiv.org/pdf/math/0404033.pdf)
[Markov Chains and Markov Chain Monte Carlo](https://www.stats.ox.ac.uk/~teh/teaching/dtc2014/Markov4.pdf)
[Genetic algorithms and Markov Chain Monte Carlo: Differential Evolution Markov Chain makes Bayesian computing easy](https://pdfs.semanticscholar.org/3aa7/ae686ecc42eeb43b4a8378f74fcf35522c55.pdf)
[Discrete-time Markov Chains and Applications to Population Genetics](https://math.rutgers.edu/images/academics/course_materials/338/14chapter4.pdf)
[Markov Chain Models of Genetic Algorithms](https://www.researchgate.net/profile/Alden_Wright/publication/2394595_Markov_Chain_Models_of_Genetic_Algorithms/links/02bfe50f61818475d1000000/Markov-Chain-Models-of-Genetic-Algorithms.pdf)
[Efficient Markov Chain Monte Carlo Methods for Global Illumination Algorithms](https://cg.informatik.uni-freiburg.de/intern/seminar/raytracing%20-%20Kraus%20-%202016%20-%20Markov%20Chain%20Monte%20Carlo.pdf)
[Efficient Synthesis of Probabilistic Programs](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/paper-5.pdf)
[On Prediction Using Variable Order Markov Models](https://web.archive.org/web/20170808205433/http://www.cs.technion.ac.il/~ronbeg/begleiter-papers/begleiter04a.pdf)
[Data Compression Using Dynamic Markov Modelling](https://web.archive.org/web/20190713091112/http://webhome.cs.uvic.ca/~nigelh/Publications/DMC.pdf)
[Markov Chains - A visual explanation](https://setosa.io/blog/2014/07/26/markov-chains/)
[Decrypting Classical Cipher Text Using Markov Chain Monte Carlo](http://probability.ca/jeff/ftpdir/decipherart.pdf)

## tagged simulation

[Characteristics of Pattern Formation and Evolution in Approximations of Physarum](https://uwe-repository.worktribe.com/OutputFile/980585)

## tagged physics

[Simulation of Clothing with Folds and Wrinkles](https://www.cs.ubc.ca/~rbridson/docs/cloth2003.pdf)

**Comment by tauoverpi** [comment]
yet to be tagged

https://ncase.me/
https://intelligence.org/files/ProblemsSelfReference.pdf
https://intelligence.org/files/DefinabilityTruthDraft.pdf
https://intelligence.org/files/ProcrastinationParadox.pdf
https://intelligence.org/files/VingeanReflection.pdf
https://ict.usc.edu/pubs/Logical%20Prior%20Probability.pdf
https://intelligence.org/files/Non-Omniscience.pdf
https://arxiv.org/pdf/1209.2620.pdf
http://lukemuehlhauser.com/wp-content/uploads/Gaifman-Concerning-measures-in-first-order-calculi.pdf
https://intelligence.org/files/QuestionsLogicalUncertainty.pdf
https://arxiv.org/pdf/1401.5577.pdf
https://intelligence.org/files/UDTSearchOrder.pdf
https://intelligence.org/files/OntologicalCrises.pdf
https://intelligence.org/files/RealisticWorldModels.pdf
https://www.nickbostrom.com/superintelligentwill.pdf

**Comment by cvoges12** [comment]
@tauoverpi Didn't think I'd see you here

**Comment by tauoverpi** [comment]
[Characteristics of Pattern Formation and Evolution in Approximations of Physarum](https://uwe-repository.worktribe.com/OutputFile/980585)

**Comment by tauoverpi** [comment]
[Towards a Physarum learning chip](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4738259/)
